tests/PHPUnit/Fixtures/OneVisitorTwoVisits.php

```diff
@@ -106,7 +106,9 @@ private function trackVisits()
 
         // Click on external link after 6 minutes (3rd action)
         $t->setForceVisitDateTime(Date::factory($dateTime)->addHour(0.1)->getDatetime());
-        self::checkResponse($t->doTrackAction('http://dev.piwik.org/svn', 'link'));
+
+        // Testing Outlink that contains a URL Fragment
+        self::checkResponse($t->doTrackAction('https://outlinks.org/#!outlink-with-fragment-<script>', 'link'));
```

Comment:
> This test would test just the tracker behavior.

The test here tests for Tracker behavior + Archiver (since both were modified in this PR) and I want to test both